cli

```diff
@@ -76,19 +76,21 @@ linkChecker(mainArgument, options, function(err, result) {
 		console.log(JSON.stringify(result, null, 2))
 	} else {
 		result.stats.errors.forEach(error => {
-			console.error(`${error.reason} from ${error.source} to ${error.target}`)
+			console.error(`${error.reason} from ${error.source}:${error.line}:${error.column} to ${error.target}`)
 		})
 		result.stats.warnings.forEach(error => {
-			console.log(`${error.reason} from ${error.source} to ${error.target}`)
+			console.log(`${error.reason} from ${error.source}:${error.line}:${error.column} to ${error.target}`)
 		})
 		console.log('')
 		console.log(`${parsedFiles} files were scanned in ${mainArgument}`)
 		console.log(`${localLinks + localAnchorLinks} local links and ${parentLinks + parentAnchorLinks} parent links and ${remoteLinks} remote links were checked`)
 		console.log(`${errors.length} errors and ${warnings.length} warnings`)
 
 	}
-   	if (parsedFiles == 0 || result.stats.errors.length > 0) {
-   		process.exit(1)
-   	}
-})
 
+	if (!process.argv.includes(`--ignore-error-exit`)) {
+		if (parsedFiles == 0 || result.stats.errors.length > 0) {
+			process.exit(1)
+		}
+	}
```

Comment:
Bad links should not automatically break a build.

Reply:
instead of process.argv we could `options.ignoreErrorExit` or the simplified version: `options.ignoreError`